src/core/queue.ts

```diff
@@ -541,7 +541,10 @@ export class Queue extends EventEmitter {
     while (failedJobs.length > 0 && start < upperLimit) {
       for (const i in failedJobs) {
         const job = failedJobs[i];
-        if (job?.backtrace[1] === "queue#forceCleanWorker") {
+        if (
+          Array.isArray(job?.backtrace) &&
+          job?.backtrace[1] === "queue#forceCleanWorker"
```

Comment:
This can be just `job?.backtrace?.[1]`.

https://www.typescriptlang.org/play?#code/PTAEHUFMBsGMHsC2lQBd5oBYoCoE8AHSAZVgCcBLA1UABWgEM8BzM+AVwDsATAGiwoBnUENANQAd0gAjQRVSQAUCEmYKsTKGYUAbpGF4OY0BoadYKdJMoL+gzAzIoz3UNEiPOofEVKVqAHSKymAAmkYI7NCuqGqcANag8ABmIjQUXrFOKBJMggBcISGgoAC0oACCoASMFmgY7p7ehCTkVOle4jUMdRLYTqCc8LEZzCZmoNJODPHFZZXVtZYYkAAeRJTInDQS8po+rf40gnjbDKv8LqD2jpbYoACqAEoAMsK7sUmxkGSCc+VVQQuaTwVb1UBrDYULY7PagbgUZLJH6QbYmJAECjuMigZEMVDsJzCFLNXxtajBdw0ABW8GkoAAvKAAN6KEolaQ9eKoMg9SD5UAAbQA5Ml4PBhQBdRQAX0UVNAOgY0EZoFp0gA-AFObBubyLFrBQBGaUITiCeDuALQeDMAAUSugAEpgiocA0PGQvIh4AMGCD2DRvm4zMx2AxmJB+LBoOpEv74HoRF4AEQAUVWDEQNX0KaSOJT4AcqGFwgAcpAJCmgioAPLfMi7QRRrSQY6oRwKVzSPCgJw+nSjLD6FAIRAw4RXYMSX3REQGDg47osNhcbhBEpAA